src/main/scala/org/hammerlab/guacamole/alignment/ReadAlignment.scala

```diff
@@ -32,7 +32,8 @@ object AlignmentState extends Enumeration {
  * @param alignmentScore Score of the alignment
  */
 case class ReadAlignment(alignments: Seq[AlignmentState],
-                         refBases: Seq[Byte],
```

Comment:
Sorry, didn't mean you delete your comment here @ryan-williams , meant to delete my empty response